sdk/core/Azure.Core.TestFramework/README.md

```diff
@@ -402,3 +402,35 @@ To download and unpack all artifacts use the `Download-DevOpsRecordings.ps1` scr
 The `Download-DevOpsRecordings.ps1` would wait for active runs to finish before retrieving artifacts unless `-NoWait` switch is used.
 
 **NOTE:** these scripts require being signed in with Azure CLI (https://docs.microsoft.com/cli/azure/authenticate-azure-cli?view=azure-cli-latest) and access to the internal DevOps project (https://dev.azure.com/azure-sdk/internal/) 
+
+## Note on sharing fields between Generated and Customized code (such as _clientDiagnostics)
+
+Some bindings require code on the customized side to access fields that are generated. For example:
+
+```csharp
+    // Generated\SparkSessionClient.cs
+    public partial class SparkSessionClient
+    {
+        private readonly ClientDiagnostics _clientDiagnostics;
```

Comment:
(extreme) nit:  You may want to close the brackets for these.   The unbalancing is making me twitch.... but that my just be me.   😄